modules/distribution/src/main/assembly/bin.xml

```diff
@@ -167,5 +167,12 @@
         <include>*</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>${project.basedir}/../../contrib/grafana</directory>
+      <outputDirectory>conf/grafana</outputDirectory>
```

Comment:
I am thinking this should show in the `contrib/grafana` in the tarball.  To me having it under `conf` makes it seems like it something you have to use inorder to run fluo.